ui/packages/consul-ui/app/components/data-loader/index.hbs

```diff
@@ -54,12 +54,14 @@
         {{#yield-slot name="disconnected" params=(block-params (component 'notification' after=(action dispatch "RESET")))}}
           {{yield api}}
         {{else}}
+        {{#if (not eq error.status '401')}}
           <Notification @sticky={{true}}>
             <p data-notification role="alert" class="warning notification-update">
               <strong>Warning!</strong>
               An error was returned whilst loading this data, refresh to try again.
             </p>
           </Notification>
+        {{/if}}
```

Comment:
If its a 401 (unauthorized) then don't show the disconnection message, just immediately try to reauth. Failure will then just show our default 403 error page.